graphene_django/filter/utils.py

```diff
@@ -11,8 +11,25 @@ def get_filtering_args_from_filterset(filterset_class, type):
     from ..forms.converter import convert_form_field
 
     args = {}
+    model = filterset_class._meta.model
     for name, filter_field in six.iteritems(filterset_class.base_filters):
-        field_type = convert_form_field(filter_field.field).Argument()
+        if name in filterset_class.declared_filters:
+            form_field = filter_field.field
+        else:
+            field_name = name.split("__", 1)[0]
+            model_field = model._meta.get_field(field_name)
+
+            if hasattr(model_field, "formfield"):
+                form_field = model_field.formfield(
+                    required=filter_field.extra.get("required", False)
+                )
+
+            # Fallback to field defined on filter if we can't get it from the
+            # model field
+            if not form_field:
```

Comment:
Hmm yeah that could work, I’ll give it a go.

Reply:
@zbyte64 actually it doesn't work because this conversion is for input arguments and so if I try and convert it from the model field duplicate types are created in the schema. I think this is the simplest solution to the bug in #678